programs/drift/src/math/position.rs

```diff
@@ -144,16 +147,87 @@ pub enum PositionUpdateType {
 pub fn get_position_update_type(
```

Comment:
would be good to have a test for this fn that covers every permutation of remainder/position direction, cant have a bug in this code

Reply:
added a ton of tests for this